test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -1,298 +1,747 @@
 use fuel_vm::fuel_crypto::Hasher;
 use fuels::{
+    contract::execution_script::ExecutableFuelCall,
     prelude::*,
     tx::{
-        field::Script as ScriptField, Bytes32, ConsensusParameters, ContractId, UniqueIdentifier,
+        field::Script as ScriptField, field::Witnesses, field::*, Bytes32, ConsensusParameters,
+        ContractId, Input as TxInput, TxPointer, UniqueIdentifier, UtxoId,
     },
 };
 use std::str::FromStr;
 
+const MESSAGE_DATA: [u8; 3] = [1u8, 2u8, 3u8];
+
 abigen!(
     TxContractTest,
     "test_artifacts/tx_contract/out/debug/tx_contract-abi.json",
 );
 
-async fn get_contracts() -> (TxContractTest, ContractId, WalletUnlocked) {
-    let wallet = launch_provider_and_get_wallet().await;
+async fn get_contracts() -> (TxContractTest, ContractId, WalletUnlocked, WalletUnlocked) {
```

Comment:
This file contains modules inside modules. It's quite a lot of scrolling to look for things. I would consider splitting up the content into files similar to how the apps repo does it

Reply:
I reeealy want to merge this before the next big set of breaking changes from the SDK lands as I've had to update these files too many times already. I'll be back in here adding more tests soon enough after the next dependency version bump, and could perform a general refactoring at that time.